src/components/Search.js

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components/macro'
+import Button from './Button'
+
+//require('dotenv').config()
+const API_KEY = process.env.REACT_APP_API_KEY
+
+export default function Search({ onSubmit }) {
+  const [options, setOptions] = useState([])
+  const [query, setQuery] = useState('')
+  const [selectedOption, setSelectedOption] = useState(null)
+  useEffect(() => {
+    setSelectedOption(null)
+    fetch(`https://api.rawg.io/api/games?key=${API_KEY}&search=${query}`)
+      .then(res => res.json())
+      .then(data => setOptions(data.results))
+  }, [query])
+  return (
+    <section>
+      <Form onSubmit={handleSubmit}>
+        <label>
+          Selected Game: {selectedOption?.name || 'None'}
+          <input
+            autoComplete="off"
+            name="gamename"
+            id="auto"
+            placeholder="Type to search"
+            value={selectedOption?.name}
+            onChange={event => setQuery(event.target.value)}
+          />
+        </label>
+        <Button>Add game</Button>
+      </Form>
+      <Suggestions className="autoContainer">
+        {query &&
+          options.map(option => {
+            return (
+              <div
+                onClick={() => setSelectedOption(option)}
+                className="option"
+                key={option.id}
+              >
+                <span>{option.name}</span>
+              </div>
```

Comment:
Should be a `button` I think. Also, you don't need the class
```suggestion
              <button
                onClick={() => setSelectedOption(option)}
                key={option.id}
              >
                <span>{option.name}</span>
              </button>
```

Reply:
Ich bin mir an der Stelle nicht sicher. Es ist ja eine autocompletion, die mir Spiele aus der API-Datenbank anzeigt, wenn ich etwas eingebe. Auf diese Ergebnisse (```<span>{option.name}</span>```), die in Form einer Liste auftauchen, kann ich dann klicken, um es ins Input einzufügen und dann mit submit oder dem "Add game"-Button als ListItem in den localStorage einfüge. ich glaube, wenn ich aus dem div nun einen button mache, könnte es verwirren wenn man den Code liest. Was meinst du dazu?

Reply:
Soweit es geht, sollten Elemente, die angeklickt werden könenn, immer `button`s sein. Vor allem aus dem Grund, dass es von Screenreadern als semantisch korrektes HTML interepretiert wird.